includes/class-wp-security-bp.php

```diff
@@ -164,6 +169,9 @@ private function define_admin_hooks() {
 		$plugin_basename = plugin_basename( plugin_dir_path( __DIR__ ) . $this->plugin_name . '.php' );
 		$this->loader->add_filter( 'plugin_action_links_' . $plugin_basename, $plugin_admin, 'add_action_links' );
 
+		// Run files test
+		$this->loader->add_action( 'wp_ajax_' . $this->plugin_name, $plugin_admin, 'run_files_test' );
```

Comment:
for better secutiry practices, 
could we use this?

`wp_doing_ajax() {
   apply_filters(...)
}`

or is it already in core?

Reply:
Yes, we should use `wp_doing_ajax()` and `is_admin()` as well but I'm still not sure when is the best moment 🤔. `apply_filters()`is to make our own hooks which can be done later.

Reply:
Added in https://github.com/cyanlove/wp-security-bp/pull/4/commits/5d51d94ca989dcc5e75628bed7737d6455a35976 but should be reviewed at some point in the future. Does it make sense to create a new issue?